index.html

```diff
@@ -226,6 +226,55 @@ <h1 class="about-title">Skills</h1>
                     </div>    
                 </div>
             </div>
-        </section>     
+        </section>   
+        <section class="form">
+            <h2 class="form-text">
+                I'm always interested in hearing about new projects,
+                so if you'd like to chat please get in touch
+            </h2>
+            <form class="form-form" action="https://formspree.io/f/xnqyvdnr" method="post">
+                <div class="input">
+                    <input class="input-x" type="text" placeholder="Full name" maxlength="50" required>
+                    <hr class="line-y">
+                </div>
+                <div class="input">
+                    <input class="input-x" type="email" placeholder="Email address" required> 
+                    <hr class="line-y">
+                </div>
+                <textarea class="textarea" placeholder="Enter Text Here" name="textarea" maxlength="500"></textarea>
```

Comment:
- [ ] [OPTIONAL] Please add the required attribute in this textarea field also, currently you can submit the form even if you leave this filed empty, as mentioned in the project requirements we have to make sure 
- A user cannot submit the form if one or more of the fields is empty.